spec/factories/user.rb

```diff
@@ -1,27 +1,30 @@
-FactoryGirl.define do
-  factory :authentication_devise_user, :class => Refinery::Authentication::Devise::User do
-    sequence(:username) { |n| "refinery#{n}" }
-    sequence(:email) { |n| "refinery#{n}@example.com" }
-    password  "refinerycms"
-    password_confirmation "refinerycms"
-  end
+begin
```

Comment:
instead, why not just check if it's `defined?(Refinery::Authentication::Devise::User)`